MAINTAINERS.md

```diff
@@ -42,17 +43,17 @@
 | JM (Jason Meridth)        | [jmeridth](https://github.com/jmeridth)                 | Approver(helm-chart) - CD, Events, Rollouts, Workflows | [GitHub](https://github.com/)                        |
 | Nicholas Morey            | [morey-tech](https://github.com/morey-tech)             | Reviewer(docs) - CD                                    | [Akuity](https://akuity.io/)                         |
 | Blake Pettersson          | [blakepettersson](https://github.com/blakepettersson)   | Approver(docs) - CD                                    | [Akuity](https://akuity.io/)                         |
-| Hari Rongali              | [harikrongali](https://github.com/harikrongali)         | Reviewer - Rollouts                                    | [Lacework](https://github.com/lacework)              |
 | Regina Scott              | [reginapizza](https://github.com/reginapizza)           | Reviewer - CD                                          | [Red Hat](https://www.github.com/redhat/)            |
 | Ishita Sequeira           | [ishitasequeira](https://github.com/ishitasequeira)     | Approver - CD                                          | [Red Hat](https://www.github.com/redhat/)            |
 | Ashutosh Singh            | [ashutosh16](https://github.com/ashutosh16)             | Approver(docs) - CD                                    | [Intuit](https://www.github.com/intuit/)             |
-| Daniel Soifer             | [daniel-codefresh](https://github.com/daniel-codefresh) | Reviewer - Events                                      | [Codefresh](https://www.github.com/codefresh/)       |
 | Isitha Subasinghe         | [isubasinghe](https://github.com/isubasinghe)           | Approver - Workflows                                   | [Pipekit](https://pipekit.io/)                       |
 | Jesse Suen                | [jessesuen](https://github.com/jessesuen)               | Lead - CD, Rollouts <br/>Approver - Workflows, Events  | [Akuity](https://akuity.io/)                         |
 | Yuan Tang                 | [terrytangyuan](https://github.com/terrytangyuan)       | Lead - Workflows <br/>Reviewer - CD                    | [Red Hat](https://www.github.com/redhat/)            |
 | William Tam               | [wtam2018](https://github.com/wtam2018)                 | Reviewer - CD                                          | [Red Hat](https://www.github.com/redhat/)            |
-| Shuangkun Tian            | [shuangkun](https://github.com/shuangkun)               | Reviewer - Workflows                                   | Alibaba                                              |
+| Shuangkun Tian            | [shuangkun](https://github.com/shuangkun)               | Approver - Workflows                                   | Alibaba                                              |
```

Comment:
We did not agree on a promotion to approver for Shangkun at this time, can you remove this one please.